tests/resourceGroups_test.go

```diff
@@ -0,0 +1,15 @@
+package tests
+
+import (
+	"testing"
+
+	"github.com/Azure/azure-sdk-for-go/services/resources/mgmt/2019-11-01/subscriptions"
+	"github.com/Azure/go-autorest/autorest"
+	"github.com/goshlanguage/cerulean"
+)
+
+func TestPutResourceGroupsRoute(t *testing.T) {
+	server := cerulean.New()
+	client := subscriptions.NewClientWithBaseURI(server.GetBaseClientURI())
+	client.Authorizer = autorest.NullAuthorizer{}
```

Comment:
```
//TODO add the test
```

Reply:
Look at me not really reading through my code before making a PR. Didn't remember I'd stubbed so much of the resource group stuff. I can fix this in this PR or we can merge the subscription stuff and I can work on a second PR for resource groups. Your call.